### PR TITLE
v5.25 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
@mkraft For v5.25 changelog, please help with details on:
 - Any new config.json settings
 - Any Database Changes
 - Any API Changes
 - Any Websocket Event Changes

**Comment by amyblais** [comment]
@mkraft Kind reminder for https://github.com/mattermost/docs/pull/3702#issuecomment-651413387

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,81 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Fixed an issue where the right-hand side overlapped the GitHub Plugin tooltip.
+ - Fixed an issue where the plugin right-hand side did not show tooltips when a user hovered over the Close or Expand/Shrink icons.
+ - Fixed an issue where query string parameters were omitted from interactive dialog request urls.
+ - Fixed an issue where ``store.GetPostsSince()`` did not sanitise deleted posts.
+ - Fixed an issue with a panic caused by nil pointer dereference in ``importTeam``.
+ 
+### config.json
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
```
Comment (amyblais) [inline source/administration/changelog.md:58]: Will update once I get details from Martin on config.json, Database Changes, API Changes and Websocket Event Changes.

**Comment by mkraft** [comment]
> @mkraft For v5.25 changelog, please help with details on:
> 
> * Any new config.json settings
> * Any Database Changes
> * Any API Changes
> * Any Websocket Event Changes

# Config

* New setting `SamlSettings.ServiceProviderIdentifier`

# DB

No changes.

# API

No changes.

# Websocket

No changes.

**Pull request merged**